bgpd/bgp_updgrp.c

```diff
@@ -703,79 +712,224 @@ static int update_group_show_walkcb(struct update_group *updgrp, void *arg)
 
 	vty = ctx->vty;
 
-	vty_out(vty, "Update-group %" PRIu64 ":\n", updgrp->id);
-	vty_out(vty, "  Created: %s", timestamp_string(updgrp->uptime));
+	if (ctx->uj) {
+		json_updgrp = json_object_new_object();
+		/* Display json o/p */
+		tbuf = time(NULL) - (bgp_clock() - updgrp->uptime);
+		json_time = json_object_new_object();
+		json_object_int_add(json_time, "epoch", tbuf);
+		json_object_string_add(json_time, "string", ctime(&tbuf));
```

Comment:
Nit: Here, may be a different meaningful object name instead of name  'string' for timer.